docs/CHANGELOG.asciidoc

```diff
@@ -51,11 +52,476 @@ Review important information about the {kib} 7.12.x releases.
 // * <<release-notes-7.0.0-alpha1>>
 
 --
+
+[[release-notes-7.13.0]]
+== {kib} 7.13.0
```

Comment:
@jen-huang @ph Noting here that the Kibana release notes have quite a few items related to Fleet and Elastic Agent. Do we duplicate these points in the release notes we provide in the Fleet User Guide, or point to these release notes? I do not like to send users on easter egg hunts, but duplication can also be problematic.